contrib/contribution-guidelines.md

```diff
@@ -1,3 +1,39 @@
 # Elite Dangerous Pilot's Network - Contribution Guidelines
 
-{{ DRAFT PLACEHOLDER - This document will hold all of the expectations of each Community Member's contributions to the Project. }}
+## What do I need to know to help?
+As our organization spans multiple subprojects, each one uses its own unique technology stack, including various programming languages, frameworks, or tools. Familiarize yourself with the specific technologies used in the subproject you're interested in by checking the README file or the project's documentation.
+
+## How do I make a contribution?
+Here's a detailed overview of how to contribute code or documentation:
+
+1. Find (or create) an issue that you are interested in addressing or a feature that you would like to add in the subproject you're focusing on. Refer to the [guidelines](#issue-creation-and-linking) for more information and check the GitHub issues page of the relevant project.
+2. Fork the repository associated with the issue to your local GitHub organization. This means that you will have a copy of the repository under `your-GitHub-username/repository-name`.
```

Comment:
Update to something like: "If you are not a registered Developer (not a part of the GitHub organization), fork the repo.... If you are a Developer, skip to step 3" ?